### ISSUE TITLE
The shape="close" button does not have an accessible name

### ISSUE THREAD

**Issue by clane** [vmware/clarity#4079]
## Describe the bug
[Screen Reader/Browser]
Any

[Description]
The shape="close" button does not have an accessible name.

See the relevant HTML below.

`<button _ngcontent-ufx-c2="" class="close ng-tns-c2-0 ng-star-inserted" type="button" style=""><clr-icon _ngcontent-ufx-c2="" class="ng-tns-c2-0" shape="close" role="none"><svg version="1.1" viewBox="0 0 36 36" preserveAspectRatio="xMidYMid meet" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" focusable="false" role="img"><path class="clr-i-outline clr-i-outline-path-1" d="M19.41,18l8.29-8.29a1,1,0,0,0-1.41-1.41L18,16.59,9.71,8.29A1,1,0,0,0,8.29,9.71L16.59,18,8.29,26.29a1,1,0,1,0,1.41,1.41L18,19.41l8.29,8.29a1,1,0,0,0,1.41-1.41Z"></path></svg></clr-icon></button>
`
[User Impact]
Screen reader users will not know what this button does.

## How to reproduce

1.  Using a screen reader, go to the Test URL
2.  Read the shape="close" button and observe that you do not hear the purpose of this button

## Expected behavior

The shape="close" button should have an accessible name

## Versions

**App**

* Angular: Unknown
* Node:  Unknown
* Clarity:  Unknown

**Device:**
Any

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes
NA

**Comment by mathisscott** [comment]
@chlane 
Where is this HTML coming from?

**Comment by chlane** [comment]
It's coming from https://clarity-light-theme-v2-a6ykys.stackblitz.io which is the Test URL. Apologies for not including it in the description.

**Comment by gnomeontherun** [comment]
This is correct if you update Clarity, that stackblitz is using 2.1 which was before #3477 was resolved. Closing as fixed.
